pynamodb/settings.py

```diff
@@ -0,0 +1,41 @@
+import imp
+import os
+from os import getenv
+from botocore.vendored import requests
+import logging
+
+log = logging.getLogger(__name__)
+
+OVERRIDE_SETTINGS_PATH = getenv('PYNAMO_CONFIG', '/etc/pynamodb/settings_override.py')
+
+settings = {}
+
+DEFAULT_REQUEST_SESSION_HEADERS = dict()
+
+default_settings_dict = {}
+default_settings_dict['REQUEST_TIMEOUT_SECONDS'] = 25
```

Comment:
i'd still really like to see all defaults in one place. currently they are in the Meta, the constructor, and here. also all should be overrideable from the settings file or Meta